vcd/resource_vcd_org_vdc.go

```diff
@@ -0,0 +1,502 @@
+package vcd
+
+import (
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+func resourceVcdOrgVdc() *schema.Resource {
+	capacityWithUsage := schema.Schema{
+		Type:     schema.TypeSet,
+		Required: true,
+		ForceNew: true,
```

Comment:
Worth adding for better validation and avoiding index out of range panics.
```
MaxItems     1,
MinItems      1
```

Reply:
Added